pkg/util/digest/digest.go

```diff
@@ -183,7 +183,7 @@ func ComputeForIntegration(integration *v1.Integration, configmaps []*corev1.Con
 			//StringData with sorted keys
 			if s.StringData != nil {
```

Comment:
Stringdata should be avoided as Kubernetes will translate all those value in the correspondent base64 in `data`. You can remove this part entirely.

Reply:
Thank you for info; also I am not sure if there shouldn't be also `binaryData` included in digest computation in case of  `Configmap`...

Reply:
BinaryData for CM would be required as well.